src/main/resources/config/fall/default.conf

```diff
@@ -29,7 +30,6 @@ root {
   run_parameters {
     outcome:  ["Acts", "Diagnoses", "Outcomes"]                               // pipeline of calculation of outcome, possible values : Acts, Diagnoses, and Outcomes
     exposure: ["Patients", "StartGapPatients", "DrugPurchases", "Exposures"]  // pipeline of the calculation of exposure, possible values : Patients, StartGapPatients, DrugPurchases, Exposures
-    hospital_stay: ["HospitalStay"] //pipeline for hospital stay, possible values :  HospitalStay
```

Comment:
why delete this config ?

Reply:
Because I don't want to run it by default.